scripts/install-azurite.sh

```diff
@@ -50,12 +49,16 @@ install_deps() {
     elif [ -n "$(command -v yum)" ]; then
       install_yum_pkgs
     fi
+
+    sudo npm install -g azurite
```

Comment:
Let's tag a specific azurite version so we don't run into a problem with a new package breaking CI in the future.

Reply:
Done, thanks.